ts/sdk/src/sdk.ts

```diff
@@ -191,12 +191,12 @@ export class BluefinProSdk {
     this.configs[Services.Exchange] = exchangeApiConfig;
     this.exchangeDataApi = new ExchangeApi(exchangeApiConfig);
 
-    const rewarsaApiConfig = new Configuration({
+    const rewardsaApiConfig = new Configuration({
```

Comment:
```suggestion
    const rewardsApiConfig = new Configuration({
```
Was this still a typo?